manim/mobject/mobject.py

```diff
@@ -1344,44 +1347,62 @@ def __init__(self, *mobjects, **kwargs):
 
 
 class _AnimationBuilder:
-    def __init__(self, mobject, overridden_animations, generate_target=True):
+    def __init__(self, mobject, generate_target=True, overridden_animation=None):
         self.mobject = mobject
-        self.overridden_animations = overridden_animations
+        self.overridden_animation = overridden_animation
         if generate_target:
             self.mobject.generate_target()
 
     def __getattr__(self, method_name):
         method = getattr(self.mobject.target, method_name)
 
-        if hasattr(method, "_animation_override"):
-            return MethodType(method._animation_override, self.mobject)
-
         def update_target(*method_args, **method_kwargs):
+            if self.overridden_animation:
+                raise NotImplementedError(
+                    "Method chaining is currently not supported for "
+                    "overridden animations"
+                )
+
```

Comment:
This could be put at the very start of `__getattr__` instead of delaying until the returned function is called

Reply:
True: 61f76a31

Reply:
You could put it even higher, before the method is even gotten, but meh